service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -16,9 +16,24 @@
 
 import java.util.Arrays;
 
+/**
+ * Lagom service implementations must create one implementation of this interface and use it to bind Service
+ * implementations.
+ *
+ * Implementors of this interface must invoke {@link ServiceGuiceSupport#bindServices(ServiceBinding[])} or
+ * {@link ServiceGuiceSupport#bindServiceInfo(ServiceInfo)} exactly-once depending on the type of Lagom service being
+ * implemented (exposing API or consume-only respsectively). These methods setup the service and may transparently add
+ * cross-cutting services like {@link MetricsService} (allows monitoring circuit-breakers from the outside).
+ */
 public interface ServiceGuiceSupport extends ServiceClientGuiceSupport {
```

Comment:
Adds restriction of invoking either `bindServices` or `bindServiceInfo` exactly once.

Introduces the idea of those methods setting up the service and what that setup may imply (adding `MetricsService`).